Core/Install/elxConversion.h

```diff
@@ -20,7 +20,7 @@
 
 #include "itkMatrix.h"
 
-#include <vnl_vector.h>
+#include <vnl/vnl_vector.h>
```

Comment:
@kaspermarstal No problem with this adjustment, bu I just wonder, does SimpleElastix directly use `elx::Conversion`? Or is this header file accidentally leaking into the SimpleElastix build, via some indirect inclusion?

Reply:
It's probably leaking, I don't think we use it directly. Will take a look at the SimpleElastix side!

Reply:
Still wondering why `#include <vnl_vector.h>` worked fine while building elastix, when it does not for SimpleElastix. But anyway, I'll just merge your PR now  😃